sycl/unittests/xpti_trace/QueueApiFailures.cpp

```diff
@@ -502,7 +507,9 @@ TEST_F(QueueApiFailures, DISABLED_QueueKernelAsync) {
 
   // Need to wait till host task enqueue kernek to check code location report.
   using namespace std::chrono_literals;
-  std::this_thread::sleep_for(10ms);
+  while (EnqueueKernelLaunchCalled != true)
+    std::this_thread::sleep_for(1ms);
```

Comment:
Would it be better to use `std::condition_variable` here instead?

Reply:
done